### PR TITLE
Update docs for `İ` and make the implementations of ẋ and İ good.

### PR DESCRIPTION
**Pull request by Lyxal**

### CODE REVIEW

**Review by razetime — APPROVED** [review]

File: vyxal/elements.py [hunk 1]
```diff
@@ -2835,8 +2842,10 @@ def repeat(lhs, rhs, ctx):
         @lazylist
         def gen():
             prev = value
+            val = value
```
Comment (pxeger) [inline vyxal/elements.py:2845]: Why are there variables called `val` and `value`? Could they be called something more descriptive please?

File: vyxal/elements.py [hunk 2]
```diff
@@ -2835,8 +2842,10 @@ def repeat(lhs, rhs, ctx):
...
             while True:
-                val = safe_apply(function, value, ctx=ctx)
+                val = safe_apply(function, val, ctx=ctx)
+                print(val)
```
Comment (pxeger) [inline vyxal/elements.py:2848]: Please avoid leaving `print`s lying around

**Review by pxeger — CHANGES_REQUESTED** [review]

File: vyxal/elements.py (hunk 2 above)
Comment (Lyxal) [reply vyxal/elements.py:2848]: Good catch - I missed that bit of debugging last night.

File: vyxal/elements.py [hunk 3]
```diff
@@ -1478,18 +1478,25 @@ def index_indices_or_cycle(lhs, rhs, ctx):
     (any, fun) -> Repeatedly apply b to a until cycle is formed, then
```
Comment (cgccuser) [inline vyxal/elements.py:1478]: Llooks like `İ` has retained the same behavior despite the change in docs, but if you're updating elements.yaml, it'd be good to change this too, for consistency.

File: vyxal/elements.py (hunk 1 above)
Comment (Lyxal) [reply vyxal/elements.py]: This has been clarified. Thank you for noticing this. :)

**Review by cgccuser — CHANGES_REQUESTED** [review]

**Review by cgccuser — APPROVED** [review]

**Review by pxeger — APPROVED** [review]

**Pull request merged**